src/effects.js

```diff
@@ -646,12 +657,12 @@ jQuery.each( {
 } );
 
 jQuery.timers = [];
-jQuery.fx.tick = function() {
+jQuery.fx.tick = function( timestamp ) {
 	var timer,
 		i = 0,
 		timers = jQuery.timers;
 
-	fxNow = jQuery.now();
+	fxNow = timestamp || getTimestamp();
```

Comment:
I'm wondering if still needs this, since this change was added for the old ipad zooming thing - https://bugs.jquery.com/ticket/12837, maybe it's not required anymore. Or should we add `Support` comment?